### PR TITLE
A separate OutputChannel for FSAC Stdout lines in DEBUG

### PR DESCRIPTION
**Pull request by adamchester**
Recently, I had a number of crashes inside `FsAutoComplete` and it was too difficult to setup the appropriate logging to dive in and get a clue about why. 

If this PR is merged, and when logging is configured with `DEBUG` level and `Console` (`OutputChannel`) enabled, all the output from `FsAutoComplete` is captured into a separate `OutputChannel`.

Additionally, there are potentially many concurrent requests to the language service, and it was difficult to correlate the logs between client (vscode) and server (FsAutoComplete). To address that, I have added the `?requestId=` query string to every request sent to the language service.

Separately from this PR, we can easily add additional logging inside the `FsAutoComplete.Suave` project or `FsAutoComplete` itself. However, for now, the only thing visible in this new output channel is as follows:
screen shot 2017-01-15 at 15 28 26

### CODE REVIEW

File: src/Core/LanguageService.fs [hunk 1]
```diff
@@ -250,23 +266,23 @@ module LanguageService =
                 let outputString = n.ToString()
                 // Wait until FsAC sends the 'listener started' magic string until
                 // we inform the caller that it's ready to accept requests.
-                let isStartedMessage = outputString.Contains ": listener started in"
+                let isStartedMessage = outputString.Contains "listener started in"
```
Comment (adamchester) [inline src/Core/LanguageService.fs]: This makes the startup detection work for Suave 2.0+ (and still works now too)

**Comment by Krzysztof-Cieslak** [comment]
Amazing, thanks.

Will you also PR logger to FSAC?

**Comment by adamchester** [comment]
Yep, will do 👍

**Comment by Krzysztof-Cieslak** [comment]
Released in `2.19.3`

**Pull request merged**